common/multipath-over-rdma

```diff
@@ -438,6 +439,18 @@ stop_soft_rdma() {
 	} >>"$FULL"
 }
 
+start_rdma() {
+	if [ -z "$use_hw_rdma" ]; then
+		start_soft_rdma
+	fi
```

Comment:
So if there is rdma hw everything is expected to be fully configured? Maybe add a meaningful msg in case the rdma hw is not fully configured? the proper place for it would be when looking for a nic by ip address...

btw currently the code falls-back to lo address, which is not something that will ever be set on an rdma nic...

Reply:
@sagi-lb Thanks for the comments.
@Kamalheib I wish the comments will help to improve this PR.